src/ContentRepository90/Legacy/NodeLegacyStub.php

```diff
@@ -21,4 +24,30 @@ public function getNodeType(): NodeType {
     public function getParent(): NodeLegacyStub {
         return new NodeLegacyStub();
     }
+
+    public function findParentNode(): NodeLegacyStub {
+        return new NodeLegacyStub();
+    }
+
+    public function findNamedChildNode(NodeName $nodeName): NodeLegacyStub
+    {
+        return new NodeLegacyStub();
+    }
+
+    public function getNode(): NodeLegacyStub {
```

Comment:
so this thing contains the signature of the original node interface and some of the "forward compatible" changes of the traversable thing ... but this is just a helper during running the migration right? These methods like all the `find*` and `get*` would be the todo to be migrated right?

Reply:
Yes, correct. This is just to enable rector to determine the return type of methods, which not exists anymore.